### PR TITLE
volume and brick dashboards are deleted when volumes are deleted

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/gluster-integration#470

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@rishubhjain @r0h4n @shtripat please review

File: tendrl/gluster_integration/message/callback.py [hunk 1]
```diff
@@ -526,9 +526,43 @@ def volume_delete(self, event):
...
+                                           ),
+                                RESOURCE_TYPE_BRICK,
+                                NS.tendrl_context.integration_id,
+                                "delete"
+                            )
+                            logger.log(
+                                "debug",
+                                NS.publisher_id,
+                                {
+                                    "message": "Update dashboard job %s "
```
Comment (r0h4n) [inline tendrl/gluster_integration/message/callback.py]: Add brick name, integration id  in log message ? Current log message is not useful

File: tendrl/gluster_integration/message/callback.py [hunk 2]
```diff
@@ -526,9 +526,43 @@ def volume_delete(self, event):
...
+                                           ),
+                                RESOURCE_TYPE_BRICK,
+                                NS.tendrl_context.integration_id,
+                                "delete"
+                            )
+                            logger.log(
+                                "debug",
+                                NS.publisher_id,
+                                {
+                                    "message": "Delete resource from graphite job %s "
```
Comment (r0h4n) [inline tendrl/gluster_integration/message/callback.py]: Add brick name, integration id in log message ? Current log message is not useful

File: tendrl/gluster_integration/message/callback.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: i will change

File: tendrl/gluster_integration/message/callback.py (hunk 2 above)
Comment (GowthamShanmugam) [reply tendrl/gluster_integration/message/callback.py]: i will add those details

**Comment by GowthamShanmugam** [comment]
@r0h4n changes done, please check

**Comment by GowthamShanmugam** [comment]
https://github.com/Tendrl/gluster-integration/issues/357

**Pull request merged**